scripts/check_docstrings.py

```
@@ -0,0 +1,63 @@
+    files = []
+    for p in root.rglob("*.py"):
+        if any(part in exclude for part in p.parts):
+            continue
+        files.append(p)
+    return files
+
+
+def check_file(path: Path) -> list[str]:
+    try:
+        src = path.read_text(encoding="utf-8")
+    except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)